app/helpers/format-percentage.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+import prettyFloat from 'code-corps-ember/utils/pretty-float';
+
+const {
+  Helper: { helper }
+} = Ember;
+
+function applyFormatting(percentage) {
+  return percentage.length == 0 ? '' : `${percentage}%`;
+}
+
+/**
+ * Helper formats a float or a string representing a float into a percentage value.
+ *
+ * By default, the value will be formatted as `'22.25%'`.
+ * If the number is an integer, the format will be `'22.00%'`
```

Comment:
It is a tad more complicated that way, but while implementing this, I found a bug, so added gain!

I was checking if there are any decimals **before** rounding. This means that

`22.001` would display as `22.00`, even though we trim decimals, because `22.001 - 22 > 0`. Now we check if there are decimals after rounding to determine if we should trim.

This means that `prettyFloat(22.001, { trimZero: true }) == '22'`, while `prettyFloat(22.006, { trimZero: true }) == '22.01'`